### ISSUE TITLE
[BUG] Text Contrast Enhanced

### ISSUE THREAD

**Issue by KatiePUX** [UKHO/admiralty-design-system#182]
Text was found to contrast with its background in a way that could be difficult for users with 
a visual impairment to perceive.

DAC Accessibility Audit outcome:
[https://ukho.sharepoint.com/sites/EXT_Data_Upload/Shared%20Documents/Forms/AllItems.aspx?OR=Teams%2DHL&CT=1711376954679&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yNDAyMjkyMDYwMiIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D&id=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta%2FDAC%20Accessibility%20Report%20WCAG%202%2E2%20UKHO%20UDU%20130324%20Final%20Version%2Epdf&viewid=8b1f0a9e%2D2897%2D4a4d%2D8283%2Dcc660e97e3e4&parent=%2Fsites%2FEXT%5FData%5FUpload%2FShared%20Documents%2FGeneral%2FAccessibility%20Audit%2FPrivate%20Beta](url)

(Page 49-50)

WCAG Reference:
1.4.6 Contrast (Enhanced) (Level AAA)
Understanding Contrast (Enhanced) | How to Meet Contrast (Enhanced)
Issue ID: DAC_Text_Contrast_Enhanced_01
URL: https://ukhodataupload.admiralty.co.uk/external/dashboard
Page title: Dashboard
Journey: 1.2

Text was found to fail the enhanced text contrast requirements. This instance can be found 
with the text found within the ‘send a survey’ component. The white text contrasts with the 
background at a ratio of 4.7:1, which suffices the minimum text contrast requirements; 
however exceeding this ratio allows more users, especially users with a visual impairment 
that affects the perception of colour to read text on the page.

**Comment by UKHOGreg** [comment]
@KatiePUX I just noticed that this issue is around level AAA compliance.

It is my understanding that we are aiming for level AA compliance.

https://github.com/UKHO/docs/blob/1a08e2cb4e2f4230c15e90f2114d38a6ee520868/quality-assurance/web-accessibility-testing.md?plain=1#L22

https://github.com/UKHO/docs/blob/1a08e2cb4e2f4230c15e90f2114d38a6ee520868/quality-assurance/accessibility-testing.md?plain=1#L27

Therefore, I believe this issue should be closed.
